metadata.json

```diff
@@ -36,15 +32,15 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.1.0 < 5.0.0"
+      "version_requirement": ">= 4.6.0 < 5.0.0"
     },
     {
       "name": "puppet/nodejs",
       "version_requirement": ">= 1.3.0 < 2.0.0"
     },
     {
       "name": "proletaryo/supervisor",
```

Comment:
No version supporting Puppet 4 explicitly, and no decent Changelog and test coverage in puppet-ghost isn't great. So, flying a bit blind and hoping for the best